backend/src/common/client/api_server/util.go

```diff
@@ -71,8 +71,7 @@ func NewHTTPRuntime(clientConfig clientcmd.ClientConfig, debug bool, tlsCfg *tls
 		host := parsedUrl.Host
 		if parsedUrl.Scheme != "" {
 			scheme = append(scheme, parsedUrl.Scheme)
-		}
-		if testconfig.ApiScheme != nil {
+		} else {
```

Comment:
you still should check if `testconfig.ApiScheme` is not nil, otherwise you'll risk hitting null deference errors